src/OpenSSL/crypto.py

```diff
@@ -1623,6 +1623,32 @@ def dump_certificate(type, cert):
     return _bio_to_string(bio)
 
 
+def dump_publickey(type, pkey):
+    """
+    Dump a public key to a buffer.
+
+    :param type: The file type (one of :data:`FILETYPE_PEM` or
+        :data:`FILETYPE_ASN1`).
+    :param pkey: The PKey to dump.
+    :type pkey: :class:`PKey`
```

Comment:
How about `:param PKey pkey: The public key to dump`?

`:class:` is definitely not necessary if you write type markup (and it's only the one type).